README.md

```diff
@@ -69,8 +69,39 @@ Faker instance used in the project (or at least an instance with the custom
 providers added to it).
 
 If using Laravel, FakerStan will automatically resolve the faker instance from
-the container using the `fake()` helper function. If not using Laravel, you can
-specify a custom factory class in the `phpstan.neon(.dist)` configuration file:
+the container using the `fake()` helper function.
+
+If not using Laravel, but you are using a framework or environment that has a PSR-11
+compliant DI container (and assuming this container is configured via a PHP file), you
+can use the PsrContainerFakerProviderFactory.
+
+```neon
+parameters:
+  fakerstan:
+    fakerProviderFactory: CalebDW\Fakerstan\PsrContainerFakerProviderFactory
+services:
+  - class: CalebDW\Fakerstan\PsrContainerFakerProviderFactory('/path/to/container.php', 'myContainer', 'FakerId')
```

Comment:
Using configuration values is definitely my preferred approach as well... honestly, I was just trying to avoid messing with the configuration schema :)

Firstly, what are your thoughts on putting those new values under a 'fakerstan > psr' node?

Either way, as soon as we define those parameters, we need some default values to be defined (or force the user to set values for them, even without any intention of using them). I'll set the default values for 'setsVariable' and 'containerFakerId' to be the same as the defaults in the constructors and I'll set the default for 'phpContainerPath' to just a generic message telling the user to set it (if it gets used).

Reply:
I thought about the same thing, although `fakerstan.psrProvider.*` might be better.

For `phpContainerPath`, you can likely just allow it to be nullable in the neon schema (setting the default to `null`)---then just throw an exception in the actual provider if the value is null.

Reply:
Does Symfony use a consistent path for the container bootstrapping? If so, then you could just use that as the default instead of `null`

Reply:
Done. Although I threw the exception in the factory before it creates the provider instead if that's OK?

Reply:
That should be fine, it just couldn't be done in the constructor as the PHPStan DI container will always construct the factory with the `null` value---but the `create` method won't be called unless the user configures it.

I'm guessing that there's not a consistent path for Symfony? I've never used it so not sure